Formula/trunk.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Homebrew formula for Trunk https://github.com/thedodd/trunk
```

Comment:
Please remove the comments.

Reply:
Will do. FWIW, in the [guide here](https://docs.brew.sh/Adding-Software-to-Homebrew#testing-and-auditing-the-formula), the suggested command `brew audit --strict --new-formula --online trunk` gave error output indicating that I should add these comments. 

I'll go ahead and remove them, but I wanted to mention that in case it might be useful info.

Reply:
```
brew audit --strict --new-formula --online trunk
trunk:
  * 1: col 1: Missing top-level class documentation comment.
  * 1: col 1: Missing frozen string literal comment.
Error: 2 problems in 1 formula detected
```

However, if the config which the brew tests use for their linter is different, then all good.

Reply:
I think you're running `brew audit --strict --new-formula --online trunk.rb` (so a file) instead of `brew audit --strict --new-formula --online trunk` (a formula within brew).

Reply:
Hmm, gives me the same output actually:
```
# Executed from the root of my fork of homebrew-core.
brew audit --strict --new-formula --online Formula/trunk.rb
trunk:
  * 1: col 1: Missing top-level class documentation comment.
  * 1: col 1: Missing frozen string literal comment.
Error: 2 problems in 1 formula detected
```

Reply:
Also, just to clarify, I was originally executing the command as `brew audit ... trunk` (the formula within brew), and then after your feedback here I executed the command pointing to the file explicitly in my repo fork and got the same output.

So, to be sure, in both cases the formula has not been merged yet (obvs), however as I did follow the instructions in the guide here (https://docs.brew.sh/Adding-Software-to-Homebrew) the formula has been placed in brew's formulae dir on my machine. So both commands are still pointing to the same `trunk.rb` file at the end of the day.

Let me know if I'm missing something. Either way, I've made the changes you've suggested.